homeassistant/components/vlc_telnet/media_player.py

```diff
@@ -46,17 +47,18 @@
 MAX_VOLUME = 500
 
 SUPPORT_VLC = (
-    SUPPORT_PAUSE
-    | SUPPORT_SEEK
-    | SUPPORT_VOLUME_SET
-    | SUPPORT_VOLUME_MUTE
-    | SUPPORT_PREVIOUS_TRACK
+    SUPPORT_CLEAR_PLAYLIST
```

Comment:
It's ok to just sort the items and rename the PR.

Reply:
Done, thanks!